app/models/organization.rb

```diff
@@ -0,0 +1,6 @@
+class Organization < ActiveRecord::Base
```

Comment:
I would suggest `Class`, but that's already taken too. It's true that the hardest part of writing software is naming things well.

You could also keep this as organization and use that term in the interface. The Classroom app manages your Organizations.